src/status_im/contexts/preview/quo/preview.cljs

```diff
@@ -46,70 +46,94 @@
   (str (humanize k) ":"))
 
 (defn- customizer-boolean
-  [{:keys [label state] :as args}]
-  (let [theme       (quo.theme/use-theme)
-        label       (or label (key->boolean-label (:key args)))
-        field-value (reagent/cursor state [(:key args)])
-        active?     @field-value]
+  [{:keys [label state set-state] :as args}]
+  (let [theme           (quo.theme/use-theme)
+        label           (or label (key->boolean-label (:key args)))
+        field-cursor    (when-not (fn? set-state)
```

Comment:
FYI: As a temporary solution until we update all component previews, we check if `set-state` is a function and use that instead of the Reagent atom. This is how we support both APIs.

I didn't bother making things pretty and copy & pasted more because most of this will go away anyway.